### PR TITLE
Fix gaslimit

### PR DESCRIPTION
**Pull request by Ashuaidehao**
Relate https://github.com/neo-project/neo/pull/2493

### CODE REVIEW

File: src/RpcServer/RpcServer.Wallet.cs [hunk 1]
```diff
@@ -228,7 +228,7 @@ protected virtual JObject SendFrom(JArray _params)
             tx.Witnesses = transContext.GetWitnesses();
             if (tx.Size > 1024)
             {
-                long calFee = tx.Size * 1000 + 100000;
+                long calFee = tx.Size * NativeContract.Policy.GetFeePerByte(snapshot) + 100000;
```
Comment (ProDog) [inline src/RpcServer/RpcServer.Wallet.cs:231]: `SendMany` and `SendToAddress`  also needs to be modified.

**Review by ProDog — APPROVED** [review]

File: src/StateService/Network/StateRoot.cs [hunk 2]
```diff
@@ -90,7 +90,7 @@ public void SerializeUnsigned(BinaryWriter writer)
 
         public bool Verify(ProtocolSettings settings, DataCache snapshot)
         {
-            return this.VerifyWitnesses(settings, snapshot, 1_00000000);
+            return this.VerifyWitnesses(settings, snapshot, 3_00000000L);
```
Comment (shargon) [inline src/StateService/Network/StateRoot.cs:93]: It's not enough with 2?

**Review by shargon — APPROVED** [review]
tested?

File: src/StateService/Network/StateRoot.cs (hunk 2 above)
Comment (ProDog) [reply src/StateService/Network/StateRoot.cs]: But it's always `MaxVerificationGas` 🤔.
https://github.com/neo-project/neo/blob/master/src/neo/SmartContract/Helper.cs#L275

**Comment by ProDog** [comment]
> tested?

Tested OK.

**Pull request merged**